### PR TITLE
Steps to migrate from prisma binding

### PR DESCRIPTION
**Pull request by LawJolla**
Here were the steps I followed to move to `Prisma Client` from `Prisma Binding`.

I know nothing about formatting the front matter for the docs, and my markdown is ability is 0.  I'm hoping I've provided the steps where someone that knows markdown can properly format it and make the appropriate front matter changes.

### CODE REVIEW

File: docs/1.17/prisma-client/migration/Migrate-from-Prisma-Bindings [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+```js
+const { Prisma } = require("prisma-binding")
+
+...
+
+const db = new Prisma({
+  resolvers,
+  typeDefs: `./generated/prisma.graphql`,
+  ...
```
Comment (brikou) [inline docs/1.17/prisma-client/migration/Migrate-from-Prisma-Bindings]: You should write `// ...` instead of `...`

Comment (brikou) [reply docs/1.17/prisma-client/migration/Migrate-from-Prisma-Bindings]: everywhere please :)

**Comment by schickling** [comment]
Thanks a lot for opening this PR @LawJolla. We're currently updating the docs to make it easier to understand when to use Prisma client vs Prisma binding. We'll get back to this PR soon!

**Comment by LawJolla** [comment]
Thanks @schickling !  I see you guys went in a different direction with the API, so I was going to close this.  But I can take a second swing once the API stabilizes.

**Comment by nikolasburk** [comment]
Hey @LawJolla, thanks so much for opening this PR and the thought you've put into it! After an internal discussion we've decided to not merge it. The main reason for that is that we see the client and bindings to be used in distinct scenarios (more info in [this](https://www.prisma.io/forum/t/help-understanding-prisma-clients-value-proposition/4394/17?u=nikolas) forum post) so that there shouldn't actually be a need to "migrate" from one approach to the other. We are working on more "official" resources (other than the linked forum post) that will make the usages of both approaches more clear!
Thanks again for taking the time to create this PR and thinking along, this is very much appreciated! 🙏

**Pull request closed without merging**